### ISSUE TITLE
Bug: Error on npm init

### ISSUE THREAD

**Issue by davelosert** [octokit/create-octokit-project.js#184]
I just tried to use this library with the command `npm init octokit-project`.

However, I run into this problem:

> const inquirer = require("inquirer");
>                  ^
> 
> Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/davelosert/.npm/_npx/b52e3113e4f00614/node_modules/inquirer/lib/inquirer.js from /Users/davelosert/.npm/_npx/b52e3113e4f00614/node_modules/create-octokit-project/create-octokit-project.js not supported.
> Instead change the require of inquirer.js in /Users/davelosert/.npm/_npx/b52e3113e4f00614/node_modules/create-octokit-project/create-octokit-project.js to a dynamic import() which is available in all CommonJS modules.
>     at Object.<anonymous> (/Users/davelosert/.npm/_npx/b52e3113e4f00614/node_modules/create-octokit-project/create-octokit-project.js:6:18) {
>   code: 'ERR_REQUIRE_ESM'
> }

Node Verison: v16.14.2
npm Version: 8.5.0

Didn't look into the details yet, but yeah, classical ESM incompatibility issue 😦.

**Comment by davelosert** [comment]
From the [Docs of Inquirer](https://www.npmjs.com/package/inquirer):

> Inquirer v9 and higher are native esm modules, this mean you cannot use the commonjs syntax require('inquirer') anymore. If you want to learn more about using native esm in Node, I'd recommend reading [the following guide](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c). Alternatively, you can rely on an older version until you're ready to upgrade your environment:
npm install --save inquirer@^8.0.0

Happy to set up a PR to do either, change this project to pure ESM (though that might cause follow up bugs) or rollback the inquirer Version to 8.x.x

**Comment by timrogers** [comment]
Personally, I think I'd try a rollback (just to hopefully unblock you 🤞) and then we can figure out ESM separately.

**Comment by davelosert** [comment]
I just tried this - unfortunately `inquirer` is not the only package but every dependency that is not `@octokit`:

- camelcase
- clipboardy
- execa
- inquirer
- spdx-license-list

They are all by [Sindre Sorhus](https://github.com/sindresorhus) who decided to push ESM forward by dropping all support on all his packages for CJS. This means: I can rollback all those packages, but they all will be pretty outdated then.

I guess the better way forward would be to try to port this library to ESM alltogether.

**Comment by timrogers** [comment]
😓 I understand the reasons for pushing ESM, but these packages are a bit of a pain!

This "create project" command doesn't do anything super magical, so what do you think of doing what it does manually, and then we can make a note to fix this as a separate issue?

**Comment by davelosert** [comment]
I already did 🙂  (see my PR over at: https://github.com/octokit/plugin-paginate-graphql.js/pull/5).
I just went ahead and copied all the stuff I needed from the rest-pagination repository.

**Comment by gr2m** [comment]
I just ran `npm init octokit-project` and it didn't throw the error for me

Did you see the issue right away or at a later step?

**Comment by gr2m** [comment]
> I guess the better way forward would be to try to port this library to ESM alltogether.

Yes I agree.

Also we should add actual tests :D A simple `node create-octokit-project.js` would have caught the error. I'm looking into it, because `npm init octokit-project` is a killer time saver if you create octokit plugins, and results in much better repositories
